### ISSUE TITLE
HTTP Request for one off should have an 'allow once' option

### ISSUE THREAD

**Issue by mralext20** [Frooxius/NeosPublic#904]
this is relating to the pop up you get when a GET or POST node is fired, NOT the explicit Request Host Access node.

basicly: some systems you may want to trust one time, rather than forever and ever. is it possible to have the existing 'Allow', aswell as a 'allow once' and 'deny' options for this?

**Comment by Frooxius** [comment]
Yeah this could be added. I'll have to make a mechanism that ensures only a single request goes though.

**Comment by Psychpsyo** [comment]
Coming from #3138, my elaboration as to what exactly I was requesting there:

> The point would be so that you can confirm every single request going to a specific host. (The popup would be valid only for confirming the single GET/POST request it was opened for)
> For websockets it would probably make the most sense to allow that one connection to get established and any other connection needs to be reconfirmed. (or the same connection if it gets interrupted and needs to reconnect.)

**Comment by AshtonSparx** [comment]
How I think it should work, it should allow 1 single pulse to trigger if you "Allow Once" and that's all it would get without another prompt, there should be a 5 second cooldown or something similar to when that prompt should come up again so it can't be spam abused.
